packages/react-native/src/config.js

```diff
@@ -3,7 +3,21 @@ const stringWithLength = require('@bugsnag/core/lib/validators/string-with-lengt
 const rnPackage = require('react-native/package.json')
 const iserror = require('iserror')
 
-const ALLOWED_IN_JS = ['onError', 'onBreadcrumb', 'logger', 'metadata', 'user', 'context', 'codeBundleId', 'plugins', 'featureFlags', 'reportUnhandledPromiseRejectionsAsHandled']
+const ALLOWED_IN_JS = [
+  'onError',
+  'onBreadcrumb',
+  'logger',
+  'metadata',
+  'user',
+  'context',
+  'codeBundleId',
+  'plugins',
+  'featureFlags',
+  'reportUnhandledPromiseRejectionsAsHandled',
+  // sendPayloadChecksums gets set in core because its 'default value' depends on other config values and so cannot remain unset
+  'sendPayloadChecksums'
```

Comment:
this doesn't seem ideal but I'm not sure there is a better way to get the "default value" behaviour of `sendPayloadChecksums` without actually setting its value when the client is created, i.e. this part:

```
    // sendPayloadChecksums is false by default unless custom endpoints are not specified
    if (!opts.endpoints) {
      opts.sendPayloadChecksums = 'sendPayloadChecksums' in opts ? opts.sendPayloadChecksums : true
    }
```

I think it would be better if the `defaultValue` part of config could produce its value based on the rest of the config but that seemed way out of scope here,